src/app/shared/services/auth.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+
```

Comment:
Should this line break be here? Decorators should be appended just about the class it defines, right? Or it does not matter with Angular JS?